CNAME

```diff
@@ -1 +0,0 @@
-read.wiki.org
```

Comment:
How about this for change flow:

feature branch =[pull]=> master =[merge]=> fork =[merge]=> gh-pages

Everyone keeps their own CNAME in their gh-pages branch. No CNAME in master.